source/docs/application-notes/tuner-evolution.rst

```diff
@@ -0,0 +1,97 @@
+Tuner and an evolution in configuration
+=======================================
+
+*Authored by Dalton*
+
+.. image:: images/tuner-evolution/tuner-v1.png
+   :alt: Image of Tuner v1
+
+Since the introduction of the CTRE Toolsuite (pre-2018), we at CTR have strived to provide intuitive means of configuring and utilizing our products. In 2018, we launched Phoenix Tuner (now lovingly referred to as Tuner v1). Tuner v1 introduced features like: batch firmware upgrading like devices, diagnostic server deployment, self tests, plotting and control.
+
+With Tuner, and by extension diagnostics, we have several primary objectives:
+
+- Ease of debugging (exposed via self test).
+- Seamless setup experience.
+- Support and integrate our extension feature-set.
+
+Tuner v1 was and is great, but we wanted to do more. In 2023, we introduced Tuner X.
+
+Introducing Tuner X
+-------------------
+
+.. image:: images/tuner-evolution/tuner-x-2023.png
+   :alt: Card overview page in Tuner X 2023
+
+The goal we had with Tuner X development was to refine and enhance the existing Tuner X feature set. We introduced `Android support <https://play.google.com/store/apps/details?id=com.ctre.phoenix_tuner>`__, improved batch upgrading, improved highlighting of duplicate devices, automatic firmware downloads (no more downloading CRFs!), improved self test and licensing support.
+
+With Tuner X, user's can:
+
+* Configure their device's name & ID
+* Blink a device, which is useful for identifying where the device is on the robot.
+* Firmware update all devices to the latest version available (no more CRF downloads).
+* Control individual motors with their Android phone, or on Windows.
+* Plot various signals such as velocity, position and yaw.
+* Self test their v6 device, which provides a marked up self test of the device.
+
+.. important:: Tuner X does not require v6 and can be used with v5 flashed devices.
+
+For a full list of features, check out the `v6 documentation <https://pro.docs.ctr-electronics.com/en/latest/docs/tuner/index.html>`__.
+
+Introducing a new iteration of Tuner X
+--------------------------------------
+
+Some of you may have noticed that your version of Tuner X has changed quite drastically. We've been working on several key improvements to the application that should dramatically improve the user experience.  While this blog will highlight some of those, it's best to just try out the new Tuner yourself.
```

Comment:
I would emphasize that their version of Tuner X has changed *recently*
So maybe "noticed _recently_ that your version of Tuner X has changed[...]" or "Tuner X has _recently_ changed quite drastically" or something along those lines